### ISSUE TITLE
Localstack some services network failure

### ISSUE THREAD

**Issue by mathijsvoss** [commandeer/open#84]
I am experiencing this issue with certain localstack services like api-gateway, sqs and sns.
I am using the commandeer version 1.4.1 on Ubuntu 20.04.1 LTS.

Using the AWS CLI i can see and create topics and queues for example.

I tried the solutions in the older issue but this is not resolving it:
https://github.com/commandeer/open/issues/7

My docker file contains the following:

```    localstack:
        container_name: "localstack"
        image: localstack/localstack-full
        networks:
            - workspace_app
        ports:
            - "4566-4599:4566-4599"
            - 9999:8080
        environment:
            - DEFAULT_REGION=eu-central-1
            - DEBUG=true
            - DOCKER_HOST=unix:///var/run/docker.sock
        volumes:
        - ./infra/localstack/data:/tmp/localstack
        - /var/run/docker.sock:/var/run/docker.sock

**Comment by fsproru** [comment]
@mathijsvoss, thanks for reporting it. Could you send us the content of your local account? You can download it from Commandeer -> Accounts -> Your local account

Also, are all your AWS endpoints set to `http://localhost:4566` in your local account section like here: https://docs.getcommandeer.com/releases/latest/version-1.2.0-6-21-2020.html#localstack ?

On a related note, did you try to start LocalStack from Commandeer?

**Comment by fsproru** [comment]
Also, seeing that you're starting LocalStack in `eu-central-1` region. This may be the reason why you're seeing network errors in Commandeer. We have the region default to `us-east-1` for local accounts. Which we're thinking to enable in a future release.

In the meantime, would you mind trying to start LocalStack in `us-east-1` by starting it in Commandeer or by specifying `DEFAULT_REGION=us-east-1` in your docker-compose file?

**Comment by mathijsvoss** [comment]
@fsproru, I tried downloading my account settings but this also triggers an error. See the screenshot below.

In my env values of Localstack i have set EDGE_PORT to 4566, i also changed the region to us-east-1.
But i have still got the same errors.

![commandeer_screenshot](https://user-images.githubusercontent.com/16712065/100431450-7a4d9880-3098-11eb-8bf0-60b7f2902064.png)

**Comment by fsproru** [comment]
Thank you. Could you try the latest version 1.5 that we just pushed? It
points Comandeer UI to http://localhost:4566 by default for local accounts.
You may need to recreate your local account once you upgrade. Let us know
how it goes.

**Comment by mathijsvoss** [comment]
I tried installing the new version but can not open commandeer?
When i try to launch the application my desktop looks to be refreshing or something like that.
You may also contact me directly by email through my license information if you prefer.

```
[mathijs:~/Downloads]$ sudo apt install ./Commandeer_1.5.0_amd64.deb 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'commandeer' instead of './Commandeer_1.5.0_amd64.deb'
The following NEW packages will be installed:
  commandeer
0 upgraded, 1 newly installed, 0 to remove and 3 not upgraded.
Need to get 0 B/75.1 MB of archives.
After this operation, 261 MB of additional disk space will be used.
Get:1 /home/mathijs/Downloads/Commandeer_1.5.0_amd64.deb commandeer amd64 1.5.0 [75.1 MB]
Selecting previously unselected package commandeer.
(Reading database ... 233998 files and directories currently installed.)
Preparing to unpack .../Commandeer_1.5.0_amd64.deb ...
Unpacking commandeer (1.5.0) ...
Setting up commandeer (1.5.0) ...
Error in file "/usr/share/applications/org.kde.kdeconnect_open.desktop": "*/*" is an invalid MIME type ("*" is an unregistered media type)
Processing triggers for mime-support (3.64ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for desktop-file-utils (0.24-1ubuntu3) ...
[mathijs:~/Downloads]$ 
```

```
[mathijs:~/Downloads]$ commandeer
11:58:07.964 › App starting...
(electron) 'getName function' is deprecated and will be removed. Please use 'name property' instead.
11:58:07.970 › Single instance lock: false
11:58:07.970 › Didn't get the single instance lock, quitting the app
11:58:08.019 › ready
(electron) The default value of app.allowRendererProcessReuse is deprecated, it is currently "false".  It will change to be "true" in Electron 9.  For more information please check https://github.com/electron/electron/issues/18397
(node:2964115) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.
(node:2964115) ProtocolDeprecateCallback: The callback argument of protocol module APIs is no longer needed.
11:58:08.076 › ready - window created. Window: true, webContents: true
11:58:08.095 › APPIMAGE env is not defined, current application is not an AppImage
```

**Comment by fsproru** [comment]
@mathijsvoss thanks for reporting it. I downloaded, installed the latest version of Commandeer 1.5.0, and launched it from the terminal.

Looking at the output, it says `Didn't get the single instance lock, quitting the app` which means that there is another `commandeer` process already running. Could you see if you have commandeer running in another terminal tab or perhaps somehow in the background?

If you run `ps ax|grep commandeer` and `kill -9 <duplicate_process_ids>` and then run `commandeer` again, it should stand up the GUI.

Let us know if the above workaround worked for you. We'll address this issue in future releases. I just noticed when you close Commandeer window on Ubuntu, it still keeps it running in the terminal.

**Comment by fsproru** [comment]
@mathijsvoss, we found the issue, working on the fix, and we'll push a fix soon.

**Comment by fsproru** [comment]
@mathijsvoss the duplicate process issue is now fixed in 1.5.1. Let us know if you're getting the network failures for LocalStack.
